.github/workflows/perplexity-maintenance.yml

```
@@ -0,0 +1,43 @@
+name: Perplexity Maintenance
+on:
+  schedule:
+    - cron: '0 7 * * 2'
+jobs:
+  maintenance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)